easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.0-GCC-12.3.0-Python-2.7.18.eb

```diff
@@ -0,0 +1,55 @@
+# Author: Maxime Schmitt, University of Luxembourg
```

Comment:
This is not needed, as Python 2.7.18 is a build dependency.

Also, it is not used as CONCOCT has `('BEDTools', '2.31.0')` as a dependency.